code/datums/quirks/negative_quirks.dm

```diff
@@ -924,12 +925,72 @@
 	. = ..()
 	var/mob/living/carbon/human/human_holder = quirk_holder
 	var/obj/item/mask_item = human_holder.get_item_by_slot(ITEM_SLOT_MASK)
-	if (istype(mask_item, /obj/item/clothing/mask/cigarette))
+	if(istype(mask_item, /obj/item/clothing/mask/cigarette))
 		var/obj/item/storage/fancy/cigarettes/cigarettes = drug_container_type
 		if(istype(mask_item, initial(cigarettes.spawn_type)))
 			quirk_holder.clear_mood_event("wrong_cigs")
-			return
-		quirk_holder.add_mood_event("wrong_cigs", /datum/mood_event/wrong_brand)
+		else
+			quirk_holder.add_mood_event("wrong_cigs", /datum/mood_event/wrong_brand)
+
+/datum/quirk/item_quirk/junkie/alcoholic
+	name = "Alcoholic"
+	desc = "You just can't live without alcohol. Your liver is a machine that turns ethanol into acetaldehyde."
+	icon = FA_ICON_WINE_GLASS
+	value = -4
+	gain_text = span_danger("You really need a drink.")
+	lose_text = span_notice("Alcohol doesn't seem nearly as enticing anymore.")
+	medical_record_text = "Patient is an alcoholic."
+	reagent_type = /datum/reagent/consumable/ethanol
+	drug_container_type = /obj/item/reagent_containers/cup/glass/bottle/whiskey
+	hardcore_value = 1
+	drug_flavour_text = "Make sure you get your favorite type of drink when you run out."
+	mail_goodies = list(
+		/obj/effect/spawner/random/food_or_drink/booze,
+		/obj/item/book/bible/booze,
+	)
+	/// Cached typepath of the owner's favorite alcohol reagent
+	var/datum/reagent/consumable/ethanol/favorite_alcohol
+
+/datum/quirk/item_quirk/junkie/alcoholic/New()
+	drug_container_type = pick(/obj/item/reagent_containers/cup/glass/bottle/whiskey,
+		/obj/item/reagent_containers/cup/glass/bottle/vodka,
+		/obj/item/reagent_containers/cup/glass/bottle/ale,
+		/obj/item/reagent_containers/cup/glass/bottle/beer,
+		/obj/item/reagent_containers/cup/glass/bottle/hcider,
+		/obj/item/reagent_containers/cup/glass/bottle/wine,
+		/obj/item/reagent_containers/cup/glass/bottle/sake)
+
+	return ..()
+
+/datum/quirk/item_quirk/junkie/alcoholic/post_add()
+	. = ..()
+	//ugh this sucks but cant use initial on lists
+	var/obj/item/reagent_containers/brandy_container = new drug_container_type()
+	favorite_alcohol = brandy_container.list_reagents[1]
+	qdel(brandy_container)
```

Comment:
why not just make an associated global list that sets this up round-start and then reference that every time rather than new a whole ass object every time, like

```dm
#define ALCOHOL_CONTAINERS list(/** alcohol containers go here */)
GLOBAL_LIST(alcohol_containers, setup_alcohol_containers())

/proc/setup_alcohol_containers()
    var/list/returnable_list = list()
    for(var/obj/item/reagent_containers/container as anything in ALCOHOL_CONTAINERS)
        container = new()
        returnable_list[container.type] = container.list_reagents[1]
        qdel(container)
    return returnable_list
```

no idea if this works and it's also space indented so good luck lol